tasks/release_from_changelog.rake

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+desc 'Generate release notes from current changelog'
+task :release_from_changelog do
+  def format_user_link(string)
```

Comment:
I was actually thinking of detecting all the user links and just appending those to the bottom of the file (what I'm manually doing for releases).

Reply:
Would you prefer a list of *all* contributors (that’s what is in the recent release notes), or only the contributors to the new version?